kategory/src/main/kotlin/kategory/data/WriterT.kt

```diff
@@ -10,29 +10,29 @@ data class WriterT<F, W, A>(val MF: Monad<F>, val value: HK<F, Tuple2<W, A>>) :
     class F private constructor()
 
     companion object {
-        inline fun <reified F, reified W, A> pure(a: A, MM: Monoid<W> = monoid(), MF: Monad<F> = monad()) =
+        inline fun <reified F, reified W, A> pure(a: A, MM: Monoid<W> = monoid(), MF: Monad<F> = kategory.monad()) =
```

Comment:
because providing default args makes the monad() method ambiguous with the one in kategory. For other types where different args are required they are not necesary because is able to determine if it comes from kategory or the same companion instance.